src/python/grpcio/grpc/_channel.py

```diff
@@ -652,16 +663,24 @@ def __init__(self, channel):
     self.managed_calls = None
 
 
-def _call_spin(state):
-  while True:
-    event = state.completion_queue.poll()
-    completed_call = event.tag(event)
-    if completed_call is not None:
-      with state.lock:
-        state.managed_calls.remove(completed_call)
-        if not state.managed_calls:
-          state.managed_calls = None
-          return
+def _run_call_spin_thread(state):
+  call_spin_running = True
+  def call_spin():
```

Comment:
`channel_spin` or `spin`, but not `call_spin` (and similarly for other names in this area).